samples/js/text_generation/beam_search_causal_lm.js

```diff
@@ -0,0 +1,34 @@
+import { LLMPipeline } from 'openvino-genai-node';
+
+main();
+
+async function main() {
+    const modelPath = process.argv[2];
+    const prompts = process.argv.slice(3);
+    
+    if (!modelPath) {
+        console.error('Please specify path to model directory\n'
+                    + 'Run command must be: `node chat_sample.js *path_to_model_dir* *prompt*`');
+        process.exit(1);
+    }
+    if (!modelPath) {
```

Comment:
```suggestion
    if (!prompts) {
```
same for samples/js/text_generation/multinomial_causal_lm.js and greedy

Reply:
Thanks!